components/Repo/Add.js

```diff
@@ -113,12 +125,17 @@ const TemplatePicker = compose(
   const templateOptions = useMemo(() => {
     return schemaOptions
       .concat(
-        (data?.reposSearch?.nodes || []).map(repo => ({
-          value: repo.latestCommit.document.meta.template,
-          text: repo.latestCommit.document.meta.title,
-          repoId: repo.id,
-          slug: repo.latestCommit.document.meta.slug
-        }))
+        (data?.reposSearch?.nodes || [])
+          .map(repo => ({
+            value: repo.latestCommit.document.meta.template,
+            text: repo.latestCommit.document.meta.title,
+            repoId: repo.id,
+            slug: repo.latestCommit.document.meta.slug
+          }))
+          .sort(
+            (repo1, repo2) =>
+              repo1.text?.toLowerCase() > repo2.text?.toLowerCase()
+          )
```

Comment:
We should always use a full fledged `compareFunction`:
```
import { ascending } from 'd3-array'
```

```suggestion
          .sort(
            (repo1, repo2) => ascending(repo1.text?.toLowerCase(), repo2.text?.toLowerCase())
          )
```
(A full fledged `compareFunction` returns `-1`, `0` and `1` instead of just `true` and `false` which will be casted to `0` and `1`, incorrectly marking lower values as equal)